CHANGELOG.ios.md

```diff
@@ -4,6 +4,17 @@
 - [iOS 2.0 SDK](http://aka.ms/gc6fex)
 - [iOS 1.2.4 SDK](http://aka.ms/kymw2g)
 
+### Version 2.2.0
+- Made Id property of records case-insensitive [328aadf](https://github.com/Azure/azure-mobile-services/commit/328aadf)
```

Comment:
This summary is a little misleading / confusing.  Better to say MSCoreData store operations now will do a case- insensitive compare on the id column.  (or some such)